go.mod

```diff
@@ -24,7 +24,6 @@ require (
 	github.com/pkg/errors v0.8.1 // indirect
 	github.com/sirupsen/logrus v1.4.2
 	github.com/spf13/afero v1.2.2 // indirect
-	github.com/spf13/cobra v0.0.5 // indirect
```

Comment:
I wonder if we should check this change in? 🤔

Reply:
Probably not. I'm still puzzled sometimes by changes in my go.mod files locally.